torch/csrc/lazy/core/lazy_graph_executor.cpp

```diff
@@ -1096,12 +1098,30 @@ std::vector<BackendDataPtr> LazyGraphExecutor::GatherTensorsData(
   return result_tensors_data;
 }
 
+void LazyGraphExecutor::TensorCollectionBarrier(SyncTensorCollection* coll) {
+  static const std::string invalid_device(
+      "Unknown0"); /* Temp solution to idetify unassigned devices */
```

Comment:
just a note for myself to remove this after https://github.com/pytorch/pytorch/pull/76517/files merged and each backend implement the default device correctly.